src/Sulu/Bundle/SecurityBundle/Command/InitCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sulu\Bundle\AdminBundle\Admin\Admin;
+use Sulu\Bundle\AdminBundle\Admin\AdminPool;
+use Sulu\Component\Security\Authentication\RoleInterface;
+use Sulu\Component\Security\Authentication\RoleRepositoryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @internal
+ */
+final class InitCommand extends Command
```

Comment:
+1 for making much more stuff final. Should that be the new default for most classes now? Or what was your reasoning for doing this? The `Builder` class above could e.g. also be `final`.

Reply:
Yeah think we should mark more classes final and internal where we don't want or can (in case of third party dependencies) provide any bc promises.